# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/verbose-init-process.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.7/init/main.c.vanilla	2024-02-17 23:10:12.102662427 +0100
+++ linux-6.7/init/main.c	2024-02-17 23:13:00.794646172 +0100
@@ -1347,7 +1347,7 @@
 	const char *const *p;
 
 	argv_init[0] = init_filename;
-	pr_info("Run %s as init process\n", init_filename);
+	printk("Running %s as init process\n", init_filename);
 	pr_debug("  with arguments:\n");
 	for (p = argv_init; *p; p++)
 		pr_debug("    %s\n", *p);
